test/tools/ossfuzz/protomutators/YulProtoMutator.h

```diff
@@ -0,0 +1,195 @@
+#pragma once
+
+#include <test/tools/ossfuzz/yulProto.pb.h>
+
+#include <libsolutil/Common.h>
+
+#include <src/libfuzzer/libfuzzer_macro.h>
+
+#include <random>
+
+namespace solidity::yul::test::yul_fuzzer
+{
+
+using ProtobufMessage = google::protobuf::Message;
+using ProtobufDesc = google::protobuf::Descriptor;
+
+template <typename Proto>
+using LPMPostProcessor = protobuf_mutator::libfuzzer::PostProcessorRegistration<Proto>;
+
+template <typename Proto>
+using FuzzMutatorCallback = std::function<void(Proto*, unsigned int)>;
```

Comment:
I'm not sure if I asked this before, but can it be `Proto const*`?

Reply:
Actually, it is intentionally not const because the callback needs write access to the pointee.

Reply:
Ok!